src/types/text_imput_v3.zig

```diff
@@ -0,0 +1,113 @@
+        pub const cursor_rectangle = 1 << 2;
+    };
+
+    pub const State = extern struct {
+        pub const Surrounding = extern struct {
+            text: ?[*:0]u8,
+            cursor: u32,
+            anchor: u32,
+        };
+
+        pub const ContentType = extern struct {
+            hint: u32,
+            purpose: u32,
+        };
+
+        pub const CursorRectangle = extern struct {
+            x: i32,
+            y: i32,
+            width: i32,
+            height: i32,
+        };
+
+        text_change_cause: u32,
+        features: u32,
+    };
+
+    seat: *wlr.Seat,
+    ressource: *wl.Ressource,
+    focused_surface: ?*wlr.Surface,
+    pending: wlr.TextInputV3.State,
+    current: wlr.TextInputV3.State,
+    current_serial: u32,
+    pending_enabled: bool,
+    current_enabled: bool,
+    active_features: u32,
+
+    link: wl.list.Link,
+
+    surface_destroy: wl.Listener(*wlr.Surface),
+    seat_destroy: wl.Listener(*wlr.Seat),
+
+    events: extern struct {
+        enable: wl.Signal(*wlr.TextInputV3),
+        commit: wl.Signal(*wlr.TextInputV3),
+        disable: wl.Signal(*wlr.TextInputV3),
+        enable: wl.Signal(*wlr.TextInputV3),
+    },
+
+    extern fn wlr_text_input_v3_send_enter(text_input: *wlr.TextInputV3, surface: *wlr.Surface) void;
+    pub fn sendEnter(text_input: *wlr.TextInputV3, surface: *wlr.Surface) void {
+        return wlr_text_input_v3_send_enter(text_input, surface);
+    }
+
+    extern fn wlr_text_input_v3_send_leave(text_input: *wlr.TextInputV3) void;
+    pub fn sendLeave(text_input: *wlr.TextInputV3) void {
+        return wlr_text_input_v3_send_leave(text_input);
+    }
+
+    extern fn wlr_text_input_v3_send_preedit_string(text_input: *wlr.TextInputV3, text: [*:0]const u8, cursor_begin: u32, cursor_end: u32) void;
+    pub fn sendPreeditString(
+        text_input: *wlr.TextInputV3,
+        text: [*:0]const u8,
+        cursor_begin: u32,
+        cursor_end: u32,
+    ) void {
+        return wlr_text_input_v3_send_preedit_string(text_input, text, cursor_begin, cursor_end);
+    }
+
+    extern fn wlr_text_input_v3_send_commit_string(text_input: *wlr.TextInputV3, text: [*:0]const u8) void;
+    pub fn sendCommitString(text_input: *wlr.TextInputV3, text: [*:0]const u8) void {
+        return wlr_text_input_v3_send_commit_string(text_input, text);
+    }
+
+    extern fn wlr_text_input_v3_send_delete_surrounding_text(text_input: *wlr.TextInputV3, before_length: u32, after_length: u32) void;
+    pub fn sendDeleteSurroundingText(
+        text_input: *wlr.TextInputV3,
+        before_length: u32,
+        after_length: u32,
+    ) void {
+        return wlr_text_input_v3_send_delete_surrounding_text(text_input, before_length, after_length);
+    }
+
+    extern fn wlr_text_input_v3_send_done(text_input: *wlr.TextInputV3) void;
+    pub fn sendDone(text_input: *wlr.TextInputV3) void {
+        return wlr_text_input_v3_send_done(text_input);
+    }
```

Comment:
there's no need for a wrapper here, just do 
```zig
pub const sendDone = wlr_text_input_v3_send_done;
```
The same goes for the other functions as well.